README.md

```diff
@@ -42,7 +42,7 @@ Build a docker image using the downloaded source code (run this in the directory
 containing the source code):
 
 ```
-docker build -t grassgis80 .
+docker build -t grassgis83 .
```

Comment:
When using this guide, this will be likely the only image of GRASS GIS you are building, no? So, maybe we don't need the version here, so just `grassgis` or something.

Reply:
Yes, make sense, will do.